modules/era/sql/guild_item_points.sql

```diff
@@ -371,21 +371,17 @@ REPLACE INTO `guild_item_points` (`guildid`, `itemid`, `rank`, `points`, `max_po
     (8, 5169, 5, 184, 2800, 2),
     (8, 4417, 5, 300, 3360, 3),
     (8, 4521, 5, 350, 3360, 3),
-    (8, 5598, 5, 400, 3600, 4),
-    (8, 5598, 5, 400, 3600, 4),
-    (8, 5599, 5, 500, 3600, 4),
-    (8, 5599, 5, 500, 3600, 4),
+    (8, 4556, 5, 504, 3840, 4),
+    (8, 4594, 5, 560, 3840, 4),
     (8, 4457, 5, 500, 3840, 5),
     (8, 4588, 5, 550, 3840, 5),
-    (8, 5598, 5, 400, 3600, 6),
-    (8, 5598, 5, 400, 3600, 6),
-    (8, 5599, 5, 500, 3600, 6),
-    (8, 5599, 5, 500, 3600, 6),
+    (8, 4556, 5, 504, 3840, 6),
+    (8, 4594, 5, 560, 3840, 6),
     (8, 4391, 5, 6, 2320, 7),
     (8, 5182, 5, 9, 2320, 7),
     (8, 4546, 6, 350, 3600, 0),
-    (8, 5572, 6, 415, 3760, 2),
-    (8, 5573, 6, 441, 3760, 2),
+    (8, 5147, 6, 375, 3680, 2),
```

Comment:
This needs to move into its own file to perform additional overrides.

Reply:
> This needs to move into its own file to perform additional overrides.

Yeah, do we have a system though for overridding other SQL files that are also in the era module? Like how can I be sure that the guild_item_points.sql in the era module will run before a cop_guild_item_points.sql also in the era module (so that the override works correctly)?

Reply:
You would need to make a new file for each era in the back_date_sqls folder and server owners decide which file to use when they setup their server. 

Also looks like you wiped out any changes in this PR, seeing zero files changed now.